labcontrol/gui/static/css/labcontrol.css

```diff
@@ -25,20 +25,18 @@ background-color: white;
 }
 
 /* Wells class */
+
+/* Commented cell (green ringed cell) */
```

Comment:
The green isn't applied to the border any more, so this should say something like "cell with green text" instead of "green ringed cell".

Reply:
Will do!